ReactiveUI/Activation.cs

```diff
@@ -37,9 +36,12 @@ namespace ReactiveUI
     public sealed class ViewModelActivator
     {
         readonly List<Func<IEnumerable<IDisposable>>> blocks;
+        readonly Subject<Unit> deactivated = new Subject<Unit>();
         IDisposable activationHandle = Disposable.Empty;
         int refCount = 0;
 
+        public IObservable<Unit> Deactivated { get { return deactivated; } }
```

Comment:
:+1: can we have an `Activated` counterpart too? Seems like we should make it easy to incorporate activation into a reactive pipeline too.